phpseclib/Common/Functions/Strings.php

```diff
@@ -295,11 +277,8 @@ public static function bits2bin($x)
 
     /**
      * Convert bits to binary data
-     *
-     * @param string $x
-     * @return string
      */
-    public static function bin2bits($x, $trim = true)
+    public static function bin2bits(string $x, $trim = true): string
```

Comment:
`$trim = true` prob ought to be changed to `bool $trim = true`

Reply:
Done